can-validate/shims/validatejs.shim.js

```diff
@@ -13,9 +13,9 @@ import validatejs from 'validate.js';
 //add shim
 function processOptions(opts){
 	//check required
-	if (opts.required) {
-		opts.presence = true;
-		if (typeof opts.required === 'object') {
+	if (typeof opts.required !== 'undefined') {
```

Comment:
Passing falsey values for the required field was causing the `required` attribute to stick around which caused validate.js to throw errors. This checks to make sure that if the value exists, we clear it.